scenes/instrument_selection/instrument_selection.gd

```diff
@@ -219,3 +219,6 @@ func _on_Items_item_rect_changed():
 
 func _on_script_changed():
 	_process_items()
+
```

Comment:
Need a double white space between functions.

Reply:
Oh it looks a little bit annoying but why not? Maybe because we always see one empty line between things and used to that.